src/main/java/com/hydratereminder/HydrateReminderPersonalityType.java

```diff
@@ -12,6 +12,7 @@
     POLITE("Polite"),
     MOTIVATIONAL("Motivational"),
     ROMANTIC("Romantic"),
+    PIRATE("Pirate");
```

Comment:
Should be a comma instead of a semicolon here.

Reply:
Ah again 🙈hope it works now😀